src/components/Vote/VotingWallet.js

```diff
@@ -165,10 +165,7 @@ function VotingWallet({
           <div className="flex align-center just-between">
             <div className="flex align-center">
               <img src={coinImg} alt="coin" />
-              <p className="content-value">
-                {getBefore(format(balance))}
-                <span>{getAfter(format(balance))}</span>
-              </p>
+              <p className="content-value">{format(balance)}</p>
```

Comment:
Do we still need `getBefore` and `getAfter` functions to be defined then?

Reply:
no need for that, i will delete these useless functions